src/theme.ts

```diff
@@ -239,7 +241,8 @@ export const theme360 = extendTheme({
           900: '#005195',
         },
         error: {
-          main: OldSchemeRed,
+          main: TestToolContrast,
```

Comment:
This should not be changed, we don't want to use the test tool colors.

Reply:
Then can we then use the color with a different name? or is this just a color we shouldn't use?